### PR TITLE
Added support for /docs/latest

### LINKED ISSUE #17: latest doc alias
as title, we should be able to do:

http://www.fastify.io/docs/latest/Getting-Started/

(this should point to the latest released version of the guide)

### PR DESCRIPTION
**Pull request by lmammino**
This is a first attempt at supporting /docs/latest.

Although I think this works for now it might make more sense to develop a slightly more complex (but powerful) feature: version aliases.

This would allow me to avoid to show in the dropdown menu `Vx.x.x` and then `latest` (as a separate version) and instead show a single entry named `Vx.x.x (latest)`.

Let me know if this is good enough for now or if you have other ideas on how to handle the aliasing.

Closes #17

### CODE REVIEW

File: src/scripts/utils/sortVersionTags.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+  if (
+    majorA > majorB ||
+    (majorA === majorB && minorA > minorB) ||
+    (majorA === majorB && minorA === minorB && patchA > patchB)
+  ) {
+    return 1
+  }
+
+  return -1
```
Comment (allevo) [inline src/scripts/utils/sortVersionTags.js:17]: Try to use this: https://www.npmjs.com/package/semver 😄

Comment (lmammino) [reply src/scripts/utils/sortVersionTags.js:17]: Lol! I didn't know it! What a waste of time re-inventing the wheel! 😆 
I will try to add this possibly tomorrow. Should be easy, thanks :)

**Comment by mcollina** [comment]
What's the status on this?

File: src/scripts/utils/sortVersionTags.js (hunk 1 above)
Comment (lmammino) [reply src/scripts/utils/sortVersionTags.js:17]: Turns out that i cannot use `semver` as i need to manage `master` and versions like `v1.2.x` which are not supported by it.

Given that I wouldn't invest more time on trying to integrate it and i would keep this function as it is for now.

**Review by allevo — APPROVED** [review]

File: src/scripts/utils/sortVersionTags.js (hunk 1 above)
Comment (allevo) [reply src/scripts/utils/sortVersionTags.js:17]: Sad... but it's ok

**Comment by lmammino** [comment]
I did a number of changes and I think this is now ready to be merged.

Latest version is never shown in versions lists, but it can be used as an alias in the urls.

Also when accessing the latest version (as `/docs/latest`) every dropdown now displays the correct latest release tag as selected.

@allevo, @mcollina, please double check if this looks like the expected behaviour to you before I proceed with the merge.

**Review by mcollina — APPROVED** [review]
LGTM

**Review by delvedor — APPROVED** [review]
LGTM

**Pull request merged**